src/main/java/teammates/common/util/Const.java

```diff
@@ -56,7 +56,7 @@
 
     public static final String ACTION_RESULT_FAILURE = "Servlet Action Failure";
 
-    public static final int SIZE_LIMIT_PER_ENROLLMENT = 150;
+    public static final int SIZE_LIMIT_PER_ENROLLMENT = 100;
```

Comment:
Why is this changed to 100? I think the 150 was intentional, to allow courses with just over 100 students to be enrolled in one shot.

Reply:
![image](https://user-images.githubusercontent.com/25312978/56465755-cefa4d80-6437-11e9-8cd9-c6bd120a6a47.png)
The enroll pages says maximum 100 students in one enrollment. Do we change it back?

Reply:
It was a hidden feature. The advertised limit remains 100. Change it back to 150 and put a comment to explain why it is 150?

Reply:
So, do we show the users maximum enroll size is 100, but if they try to enroll students slightly more than 100, we still allow it?

Reply:
@damithc the bigger question is, why does it become a "hidden" feature?

Reply:
100 is a good round number and easy for users too e.g., when dividing a spreadsheet into batches.
However, it turns out that it is fairly common for instructors to have just over 100 students and also fairly common for them to not see the limit and try to enroll all at once. Allowing a buffer of 50 more allow such requests to go through -> reduces annoyance to users and saves us some requests too. This is a bit similar to having a hidden buffer for submission deadlines. 
Having said that, I just realized this hidden feature is no longer usable because the spreadsheet limits the number of rows to 100. Which means we can go back to 100. Just make sure there is no off-by-one error. i.e., we support 100, not 99, students at one go (I remember having a bug like that once)

Reply:
OK